scripts/provisioning/vmhost/gci-create-cluster.sh

```diff
@@ -232,7 +232,7 @@ clone_hare()
 
     print_msg "Executing [git checkout -b " \
         "$HARE_USER/hare-$HARE_USER_BRANCH FETCH_HEAD]"
-    git checkout -b $HARE_USER/hare-$HARE_USER_BRANCH FETCH_HEAD
+    git checkout -b "$HARE_USER"/hare-"$HARE_USER_BRANCH" FETCH_HEAD
```

Comment:
it's better to have:
git checkout -b "$HARE_USER/hare-$HARE_USER_BRANCH" FETCH_HEAD

Reply:
done